### PR TITLE
LTV provider: Add debug logs when discarding title

### PR DESCRIPTION
**Pull request by fernandog**
@ratoaq2 with the changes you asked. If is ok, I will also PR upstream

### CODE REVIEW

File: medusa/subtitle_providers/legendastv.py [hunk 1]
```diff
@@ -436,7 +445,7 @@ def query(self, language, title, season=None, episode=None, year=None):
                 for r in releases:
                     subtitle = LegendasTVSubtitle(language, t['type'], t['title'], t.get('year'), t.get('imdb_id'),
                                                   t.get('season'), a, r)
-                    logger.debug('Found subtitle %r', subtitle)
+                    logger.info('Found subtitle %r', subtitle)
```
Comment (ratoaq2) [inline medusa/subtitle_providers/legendastv.py]: This is the only thing. That should still be debug in subliminal world. 
All the rest is great 👍

Comment (fernandog) [inline medusa/subtitle_providers/legendastv.py]: Ok thanks

**Comment by fernandog** [comment]
@ratoaq2 approve?

**Comment by fernandog** [comment]
@ratoaq2 btw we need to revert the level mappigns for ltv and itasa or proper fix it as we told you in slack

**Pull request merged**